apps/prairielearn/src/pages/administratorFeatures/administratorFeatures.html.ts

```diff
@@ -0,0 +1,169 @@
+// @ts-check
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { FeatureGrantRow } from './administratorFeatures';
+
+interface AdministratorFeaturesProps {
+  features: string[];
+  resLocals: Record<string, any>;
+}
+
+interface AdministratorFeatureProps {
+  feature: string;
+  featureGrants: FeatureGrantRow[];
+  resLocals: Record<string, any>;
+}
+
+interface FeatureGrantProps {
+  featureGrant: FeatureGrantRow;
+}
+
+export function AdministratorFeatures({ features, resLocals }: AdministratorFeaturesProps) {
+  return html`
+    <!DOCTYPE html>
+    <html>
+      <head>
+        ${renderEjs(__filename, "<%- include('../partials/head'); %>", resLocals)}
+      </head>
+      <body>
+        ${renderEjs(__filename, "<%- include('../partials/navbar'); %>", {
+          ...resLocals,
+          navPage: 'admin',
+          navSubPage: 'features',
+        })}
+        <main id="content" class="container">
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">Features</div>
+            ${features.length > 0
+              ? html`<div class="list-group list-group-flush">
+                  ${features.map((feature) => {
+                    return html`
+                      <div class="list-group-item d-flex align-items-center">
+                        <a
+                          href="${resLocals.urlPrefix}/administrator/features/${feature}"
+                          class="mr-auto text-monospace"
+                        >
+                          ${feature}
```

Comment:
```suggestion
                          ${AdministratorFeature({feature, featureGrants, resLocals})}
```

Reply:
Why? `AdministratorFeature` is a top-level page, we don't want to render it inline in _another_ page.